react/official/sidemenu/src/components/Menu.tsx

```diff
@@ -75,7 +75,7 @@ const Menu: React.FC = () => {
             return (
               <IonMenuToggle key={index} autoHide={false}>
                 <IonItem className={location.pathname === appPage.url ? 'selected' : ''} routerLink={appPage.url} routerDirection="none" lines="none" detail={false}>
-                  <IonIcon slot="start" ios={appPage.iosIcon} md={appPage.mdIcon} />
+                  <IonIcon aria-hidden={true} slot="start" ios={appPage.iosIcon} md={appPage.mdIcon} />
```

Comment:
```suggestion
                  <IonIcon aria-hidden="true" slot="start" ios={appPage.iosIcon} md={appPage.mdIcon} />
```

`aria-hidden` is an attribute not a property, so it accepts the string `"true"` not the boolean.